configuration/initialization.sh

```diff
@@ -50,10 +127,19 @@ echo \
   sudo tee /etc/apt/sources.list.d/docker.list > /dev/null
 sudo apt-get update
 sudo apt-get -y install docker-ce docker-ce-cli containerd.io docker-buildx-plugin docker-compose-plugin
-sudo docker compose up -d
+
+# Check if Port Drayage functionality is enabled
+if [[ $PORT_DRAYAGE_ENABLED == "TRUE" ]]; then
+    sudo docker compose up -d
+elif [[ $PORT_DRAYAGE_ENABLED == "FALSE" ]]; then
+    sudo docker compose up php -d
+else
+    echo "Invalid value for PORT_DRAYAGE_ENABLED. Please provide either TRUE or FALSE."
+    exit 1
+fi
 
 # Create V2X Hub user
-cd $mysqlDir
+cd "$mysqlDir" || return
```

Comment:
Why is this necessary?

Reply:
Added a comment next to return saying "# return in case cd fails"